documentation/release_notes.markdown

```diff
@@ -4,6 +4,49 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.1
+
+Released June 18, 2015
+
+This is a security and bug fix release of Puppet Server 1.1 series.  No new
+features have been added since 1.1.0 in accordance with [Semantic
+Version](http://semver.org) guidelines.  All users are recommended to upgrade.
+
+### Bug fixes
+
+#### Upgrade JRuby from 1.7.20 to 1.7.20.1 to resolve CVE-2015-4020
+
+Bump JRuby dependency to version 1.7.20.1 to resolve
+[CVE-2015-1855](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-1855)
+where JRuby has updated Rubygems to version 2.4.8. Rubygems 2.4.8 addresses
+CVE-2015-1855 to resolve some problems with wildcard matching of hostnames. See
+ruby-lang.org’s description for more info.
+
+ * [SERVER-761](https://tickets.puppetlabs.com/browse/SERVER-761)
+
+#### Consolidate environment handling behavior
+
+Consolidate JRuby environment handling, which was previously inconsistent across
+the use cases of `puppetserver gem`, `puppetserver irb`, `puppetserver ruby` and
+the puppetserver service.
+
+ * [SERVER-297](https://tickets.puppetlabs.com/browse/SERVER-297)
+
+#### Return good Content-Type for CA errors
+
+Two bugs in the way the CA service responds which caused issues for services
```

Comment:
Maybe delimit the "which caused issues for services consuming the CA API" with -- or parentheses to make it a little easier to read the sentence?